x-pack/auditbeat/module/system/user/_meta/fields.yml

```diff
@@ -24,6 +24,7 @@
       type: keyword
```

Comment:
I think you mean `user_information`? I don't have a strong opinion I think. It's filled with strings like `systemd Network Management,,,`, `Gnats Bug-Reporting System (admin)`. I don't expect too many users searching directly on it rather than something more unique like `name`.

Reply:
yes, not sure how the comment added up here :-(

Is the number of different strings limited? Do you expect users to group on these?

Reply:
When in doubt, use `keyword`. `text` should be added as a multi-field if deemed necessary.

Reply:
> `text` should be added as a multi-field if deemed necessary.

Can Beats generate multi-fields?

Reply:
yes:

```
    - name: thread
      type: keyword
      description: >
        Information about the running thread where the log originate.
      multi_fields:
        - name: text
          type: text
```